setup.cfg

```diff
@@ -38,7 +38,7 @@ exclude=
 max-line-length=90
 #ignore=W291,E303,W391,F403,F401,W503,W1202
 select = C,E,F,W,B,B950
-ignore = E501,W503,E203,W201
+ignore = E501,W503,E203,W201,E712
```

Comment:
Yes, because it complains about numpy arrays being compared to True or False elementwise and made it impossible for me to check in already existing code in a completely unrelated code area.

Reply:
I will try to fix it in another PR (there is no real point in doing this for numpy arrays either, better use `np.any` or `np.all` or whatever is equivalent.

But I hate that these pre-commit hooks force you to do changes in unrelated sections of the files. They should only look at the diff, not at the whole file...